Consent or pay (#1901)
guardian/csnx: .changeset/breezy-drinks-live.md

---
'@guardian/libs': major
---

> Add the ability for consent or pay by adding two new params to cmp.init

Author: Add more info

---
'@guardian/libs': major
---

> 

Reviewer: ```suggestion
#### Changes to enable consent or pay in the CMP
```

> - Add the ability for consent or pay by adding two new params, useNonAdvertisingList and isUsedSignedIn to cmp.init

Reviewer: How do you enable consent or pay? Should we describe that here or link to some docs?

- Using Sourcepoint Subdomain property if useNonAdvertisedList is true

> - Merging user consent from Advertising to Non-Advertising vendor list

Reviewer: How does this affect a consumer of libs / CMP?

Author: Thanks, this has been updated

> - Redirecting user to Support when clicking Reject for Consent or Pay users

Reviewer: ```suggestion
- The CMP will now redirect to https://support.theguardian.com/guardian-ad-lite when clicking Reject for Consent or Pay users
```

> - Extending \_\_tcfapi to postCustomConsent as part of merging user consent process.

Reviewer: How does this affect consumers of libs / CMP

Author: I expanded on this in 'Merging user consent from Advertising to Non-Advertising vendor list'
